STM32Resources/install.sh

```diff
@@ -17,10 +34,18 @@ if ! type "st-flash" > /dev/null; then
     ln -s /opt/stlink/build/st-flash /usr/local/bin/st-flash
     ln -s /opt/stlink/build/st-info /usr/local/bin/st-info
     ln -s /opt/stlink/build/src/gdbserver/st-util /usr/local/bin/st-util
-    echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:/opt/stlink/build" >> ~/.profile
-    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/stlink/build
-    echo "export PATH=\$PATH:/opt/stlink/build" >> ~/.profile
-    export PATH=$PATH:/opt/stlink/build
+    if [ "$(. /etc/os-release; echo $NAME)" = "Ubuntu" ]; then
+        echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:/opt/stlink/build" >> ~/.profile
```

Comment:
For all these if else statements, it could be simplified by just setting a variable for the `bash_profile` file.

```
if Ubuntu, then
    BASH_PROFILE="~/.profile
elseif Fedora, then
    BASH_PROFILE=~/.bash_rc
fi
```
Something like that. Then just reference that variable for all your other calls.